index.html

```diff
@@ -651,6 +651,9 @@ <h3>Controllers</h3>
 certain [=verification methods=] for specific purposes. If the `controller`
 property is not present, the value expressed by the `id` property MUST be
 treated as if it were also set as the value of the `controller` property.
+If the `controller` property is present, it establishes one or more
+[=controllers=] that have exclusive change control over the document.  This set MAY include
```

Comment:
> You can use a pattern of self-reference to imply that the VDR has control, but then it is NOT the value of the controller property who has control, but the VDR itself.

Despite being true, this part feels like a bit of a red-herring to me. The VDR is always the root of control and must therefore consent to anything expressed in a DID document, such as what appears in any property, including the `controller` property. Whether updates for a particular DID method are performed with the use of verification methods in the DID document itself (or those associated with a `controller`) or via some other mechanism doesn't change this.

Since applications only work at the DID document layer, they must be able to rely on information expressed there for whatever their needs may be -- they must not be required to break the abstraction and understand the details of the VDR, etc.

Reply:
> There are several DID methods for which it is NOT possible to definitively specify, in a single URL, who has exclusive change control over the document.

I do not read `one or more [=controllers=] that have exclusive change control` to say that "a single URL" is used to specify this set of controllers; rather, that `controllers` can have one or more values, which together comprise the set of entities that control the DID.

> this statement reifies the exact confusion others are reading into the spec

I don't think anyone's "reading [confusion] into the spec." Rather, I think that this "confusion" indicates that the spec doesn't say what you're suggesting it does.

The meaning you describe here may have been your intent when it was written, but my understanding has never been that the `controller` property was meant to lead to DID documents that were to be interpolated into the DID document that contained that `controller` value.

I have always understood `controller` to identify one or more entities that were permitted to cause changes to that containing DID document, just as `subject` identifies one or more entities for which that DID document contains interaction information. Those entities might be identified by DIDs, but the point of using those DIDs in this way was only for such identification.

Reply:
This is an interesting issue. To me, control over who can changes the DID document has always been DID method specific. While, it makes sense that the controller property identifies entities authorized to mutate the DID document. 

In practice, how is this realised today? Which DID methods are requiring some form of authorization proof over an update submitted to a DID. And which resolution processes are verifying that proof against the controller property in the DID document?

Also @TallTed 
> my understanding has never been that the controller property was meant to lead to DID documents that were to be interpolated into the DID document that contained that controller value.

I believe this section below of the spec is where the spec defines that controller properties should be interpretted in this way.

When a controller property is present in a [controller document](https://w3c.github.io/controller-document/#dfn-controller-document), its value expresses one or more identifiers. Any [verification methods](https://w3c.github.io/controller-document/#dfn-verification-method) contained in the [controller documents](https://w3c.github.io/controller-document/#dfn-controller-document) for those identifiers SHOULD be accepted as authoritative, such that proofs that satisfy those [verification methods](https://w3c.github.io/controller-document/#dfn-verification-method) are considered equivalent to proofs provided by the [subject](https://w3c.github.io/controller-document/#dfn-subjects).

Reply:
Perhaps we can discuss the different means of "control" evident in this discussion on the next working group call, @brentzundel ?